.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,69 @@
+name: 🚀 Feature request
+description: Suggest an idea for this project 💡
+labels:
+  - enhancement
+
+body:
+  - type: textarea
+    id: summary
+    attributes:
+      label: Summary
+      value: |
+        **As** "role name"\
+        **I want** "a feature or functionality"\
+        **So that** "I get certain business value"
+    validations:
+      required: false
```

Comment:
Surely at least the summary should be required?

Reply:
I'm not too sure about that, but I could live with either or. My reasoning is that for a very simple feature request issue it might not make sense to state e.g. '**As** developer **I want** this spelling error to be corrected **so that** I can read the text easier', but rather just provide such info in the `context` field. But sure, I could change to required here.